packages/components/ovh-shell/src/client/index.ts

```diff
@@ -70,6 +70,7 @@ export default function init(applicationId: ApplicationId) {
   }
 
   return initPromise.then((shellApi) => {
+    shellApi.ux.enableAccountSidebarToggle();
```

Comment:
does it changes anything ? it looks like it's already enabled by default, see :
https://github.com/ovh/manager/blob/d2632b631c401eb875f8cb5e5daf34ddfb072655/packages/components/ovh-shell/src/plugin/ux/components/sidebar.ts#L14